src/net.cpp

```diff
@@ -1856,28 +1856,32 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
             }
         }
 
-        // Feeler Connections
```

Comment:
Can this probably migrate to the enum?

Reply:
Partially done in 630b9a5e1872ac91949b1e30f8cd7f28d1c946ab, but some of the details are lost.

Reply:
what details are lost? the only thing not included is "Start attempting feeler connections only after node finishes making outbound connections," but the ordering of connections is captured in `ThreadOpenConnections`. this seems like the same point as addressed over in [this thread](https://github.com/bitcoin/bitcoin/pull/19724#discussion_r475145927) where you've agreed with me??